mpas_analysis/ocean/ocean_modelvsobs.py

```diff
@@ -93,11 +93,11 @@ def ocn_modelvsobs(config, field):
 
     elif field.lower() == 'sst':
 
-        ds = xr.open_mfdataset(infiles, preprocess=lambda x: preprocess_mpas(x, yearoffset=yr_offset,
-		                    timeSeriesStats=True,
-		                    timestr='time_avg_daysSinceStartOfSim',
-		                    onlyvars=['time_avg_activeTracers_temperature'],
-		                    selvals={'nVertLevels':1}))
+        ds = xr.open_mfdataset(infiles, preprocess=lambda x: 
+                               preprocess_mpas(x, yearoffset=yr_offset,
+                                               timestr=['xtime_start', 'xtime_end'],
+		                               onlyvars=['time_avg_activeTracers_temperature'],
+		                               selvals={'nVertLevels':1}))
         ds = remove_repeated_time_index(ds)
         ds.rename({'time_avg_activeTracers_temperature':'mpasData'}, inplace = True)
 
```

Comment:
@xylar: this is what I was referring to in my comments to #48. But I guess I was wrong, because xtime_start and xtime_end are _already_ supported in alpha.